### ISSUE TITLE
Unable to set or get localForage

### ISSUE THREAD

**Issue by richie777cfc** [thgreasi/localForage-cordovaSQLiteDriver#31]
Hi, 

I set up localForage as mentioned in your docs and I'm able to set sqlite engine to use with localForage getting cordovaSQLiteDriver in output.  But when I'm trying to set or get localForage item I'm only getting output as 

Promise {status: "pending"} = $1

**Comment by thgreasi** [comment]
Have you tried the two demos from the README?
Can you provide a bit more context? Library versions & some code?

**Comment by richie777cfc** [comment]
hi,
I used command line phonegap to build my project. A basic project with example html that you get when you create a phonegap project with command line. I then used the commands
`phonegap plugin add cordova-sqlite-storage`
`npm i --save localforage localforage-cordovasqlitedriver`
`phonegap plugin add cordova-sqlcipher-adapter --save`

After doing this I still wasn't able to get any output from the sample javascript code you have provided:
`<script>
    localforage.defineDriver(window.cordovaSQLiteDriver).then(function() {
        return localforage.setDriver([
            // Try setting cordovaSQLiteDriver if available,
          window.cordovaSQLiteDriver._driver,
          // otherwise use one of the default localforage drivers as a fallback.
          // This should allow you to transparently do your tests in a browser
          localforage.INDEXEDDB,
          localforage.WEBSQL,
          localforage.LOCALSTORAGE
        ]);
    }).then(function() {
      // this should alert "cordovaSQLiteDriver" when in an emulator or a device
      alert(localforage.driver());
      // set a value;
      return localforage.setItem('testPromiseKey', 'testPromiseValue');
    }).then(function() {
      return localforage.getItem('testPromiseKey');
    }).then(function(value) {
      alert(value);
    }).catch(function(err) {
      alert(err);
    });
    </script>`

So I then picked up the `localforage.js` and `localforage-cordovasqlitedriver.js` from your sample project and included them manually in the project.
After doing this I was able to set the driver as sqllite driver and I got output as cordovaSQLiteDriver in the prompt when I launch the app in the simulator (using xcode). But no prompt after that showing me the localForage retrieved value.
I haven't tried the demo yet and I will be trying it now after this comment.
Still, do let me know if i missed something while setting up my project to resolve this problem.

**Comment by thgreasi** [comment]
Hey any status update on this?

**Comment by richie777cfc** [comment]
Hey. Sorry I was out on a vacation. 
I tried the demos and surprisingly they worked. I was able to successfully use local forage. When I tried to follow the steps you mentioned in read me for installation, that didn't work so I copied over two local forage js files from the demo project and it started working as well. Probably something's wrong with the npm way. I tried several times but couldn't get it to work until I manually added the js files.
